src/theme.js

```diff
@@ -1,3 +1,13 @@
 const { default: CreateMuiTheme } = require('@material-ui/core/styles/createMuiTheme');
+const { default: amber } = require('@material-ui/core/colors/amber');
 
```

Comment:
_Slightly_ abusing the GitHub suggestion feature to add a comment that @wswoodruff mentioned would be beneficial. It seems apt to include that in this PR.

```suggestion

// The object below overrides or extends the default material-ui theme, which can currently be found here: https://material-ui.com/customization/default-theme/#default-theme

// Any of the values in that default theme may be used throughout this project, even if they aren't explicitly defined here (e.g., `theme.shape.borderRadius` will return 4, even though `shape` isn't currently defined in this file)

```